mm2src/rpc/dispatcher/dispatcher_legacy.rs

```diff
@@ -60,6 +61,7 @@ pub fn dispatcher(req: Json, ctx: MmArc) -> DispatcherRes {
         // Sorted alphanumerically (on the first latter) for readability.
         // "autoprice" => lp_autoprice (ctx, req),
         "active_swaps" => hyres(active_swaps_rpc(ctx, req)),
+        "add_node_to_version_stat" => hyres(add_node_to_version_stat(ctx, req)),
```

Comment:
Please move the new RPCs to `mmrpc:2.0`, the `dispatcher_legacy` is used to keep backward compatibility for older methods. All the new ones should follow the 2.0 protocol.